ansible_runner/display_callback/module.py

```diff
@@ -498,6 +520,7 @@ def v2_runner_on_start(self, host, task):
             host=host.get_name(),
             task=task
         )
+        self._host_start[host.get_name()] = current_time()
```

Comment:
@matburt @wenottingham,

This doesn't work on Ansible pre-2.8, because `runner_on_start` was introduced in 2.8:

https://github.com/ansible/ansible/pull/47684/